lib/workload/stateless/stacks/sample-sheet-check/sample-sheet-check-lambda/tests/sample/mock-2.csv

```diff
@@ -19,5 +19,5 @@ AdapterRead2,AGATCGG,,,,,,,,,,
 ,,,,,,,,,,,
 [Data],,,,,,,,,,,
 Lane,Sample_ID,Sample_Name,Sample_Plate,Sample_Well,Index_Plate_Well,I7_Index_ID,index,I5_Index_ID,index2,Sample_Project,Description
-3,MDX200001_L2000001,L0000000_topup,,,ABCD,ABCD,ABCD,ABCD,ABCD,ABCD,ABCD
-1,MDX200001_L2000002,L0000000_topup,,,ABCD,ABCD,ABCD,ABCD,ABCD,ABCD,ABCD
+3,MDX200001_L2000001,L2000001_topup,,,ABCD,ABCD,ABCD,ABCD,ABCD,ABCD,ABCD
```

Comment:
So to clarify 

This would be okay

```
MDX200001_L2000001_topup,L2000001_topup
```

But this is not okay?

```
MDX200001_L2000001,L2000001_topup

```

Reply:
It might be me putting the test data inconsistently, but if `MDX200001_L2000001_topup,L2000001_topup` is what we aiming for, I'll add a patch